src/components/productDetails/card.jsx

```diff
@@ -55,16 +56,18 @@ const Card = ({
             )}
           </button>
           <figure className="image is-4by3">
-            <img
-              name={product.id}
-              onClick={onClickDetails}
-              src={
-                productImage === null
-                  ? "https://image.shutterstock.com/image-vector/no-image-available-sign-absence-260nw-373243873.jpg"
-                  : productImage
-              }
-              alt="Related-Product Item"
-            />
+            <AttachProductLink listName={listName} productId={product.id}>
+              <img
+                name={product.id}
+                onClick={onClickDetails}
```

Comment:
This onClick event may be redundant now that you have a Link around these images

Reply:
Nice catch. I will definitely be removing this in the future.